### PR TITLE
Replacing Dex with Ory Hydra login and consent app in api-gateway tests

### PR DESCRIPTION
**Pull request by cnvergence**
**Description**

Changes proposed in this pull request:

- Since Kyma 2.0 won't include Dex, we need to change our JWT tests in api-gateway.

**Related issue(s)**

See also #11493

### CODE REVIEW

File: tests/integration/api-gateway/gateway-tests/manifests/k8s/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      version: v1
+  template:
+    metadata:
+      labels:
+        app: ory-hydra-login-consent
+        version: v1
+    spec:
+      containers:
+        - name: login-consent
+          image: piotrkpc/hydra-login-consent:test_0.1
```
Comment (Tomasz-Smelcerz-SAP) [inline tests/integration/api-gateway/gateway-tests/manifests/k8s/deployment.yaml:19]: We can't use such image. We have to provide something more "official". Perhaps a little project in kyma-incubator.

File: resources/ory/values.yaml [hunk 2]
```diff
@@ -152,6 +152,7 @@ oathkeeper:
           config:
             jwks_urls:
             - http://dex-service.kyma-system.svc.cluster.local:5556/keys
```
Comment (Tomasz-Smelcerz-SAP) [inline resources/ory/values.yaml:154]: Is dex still required?

File: tests/integration/api-gateway/gateway-tests/manifests/k8s/deployment.yaml (hunk 1 above)
Comment (cnvergence) [reply tests/integration/api-gateway/gateway-tests/manifests/k8s/deployment.yaml:19]: Currently working on it. Here is the image for testing that pipelines work.

File: resources/ory/values.yaml (hunk 2 above)
Comment (cnvergence) [reply resources/ory/values.yaml:154]: I thought that I would leave it in Ory for now - for the sake of old Kyma versions, where Dex is deployed

**Review by mjakobczyk — APPROVED** [review]

**Pull request merged**